### DISCUSSION TITLE
local tunnel custom GPT method, unable to connect smart chat

### DISCUSSION THREAD

**Discussion by princinv** [brianpetro/smart-connect#53] [Help & Support]
i feel like i am so close on this, i followed the [guide](https://docs.smartconnections.app/Smart-Connect-app/Creating-a-Custom-GPT-with-Smart-Connect) in the docs.

I have the model up and running and connected to the smart connect app  i am just not sure how i am supposed to integrate it with smart chat. i have tried both the local tunnel url with "default" api key, and i have tried the chatgpt.com url with an openAI api key, but i must be doing something wrong here. 

any tips?

thanks in advance.

**Comment by brianpetro** [comment]
The Smart Chat isn't quite ready for the Smart Connect app yet. 

So you're limited to using it with ChatGPT currently.

But Smart Chat will be builtin to the Smart Connect app soon 🌴

**Reply by bloominburges** [reply]
Sorry so is this video still valid to follow? 
https://youtu.be/niX9U8znJAo?si=9f2m3Tr22lnzEirW

**Reply by brianpetro** [reply]
This is newer but only for Smart Connect (not the Obsidian plugin) https://youtu.be/9vIy_tKX6S8?si=0dxKtjciJowmzX5F

The Obsidian plugin Smart Connections pane should "just work" after installing the plugin and allowing the embedding process to be completed.

The Smart Chat requires configuring a chat model using the gear icon in the top-right corner of the Smart Chat 🌴

**Comment by princinv** [comment]
thanks, man.
